packages/client/tests/functional/tracing/tests.ts

```diff
@@ -426,8 +440,14 @@ testMatrix.setupTestSuite(
                 children: [
                   engineConnection(),
                   ...txQueries,
-                  { name: 'prisma:engine:itx_query_builder', children: [...createDbQueries(false), engineSerialize()] },
-                  { name: 'prisma:engine:itx_query_builder', children: [findManyDbQuery(), engineSerialize()] },
+                  {
+                    name: 'prisma:engine:itx_query_builder',
+                    children: [...createDbQueries(false), engineSerializeQueryResult()],
+                  },
+                  {
+                    name: 'prisma:engine:itx_query_builder',
+                    children: [findManyDbQuery(), engineSerializeQueryResult()],
+                  },
```

Comment:
No `engineSerializeFinalResponse()` here because of https://github.com/prisma/prisma/issues/21402.